### PR TITLE
DDF-3511 Export Atom package for downstream dependencies

### PR DESCRIPTION
**Pull request by bdeining**
#### What does this PR do?
Export Atom package for downstream dependencies
#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
@GabrielFabian @mackncheesiest 
#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Ask 2 committers to review/merge the PR and tag them here. If you don't know who to ask, you can request reviews in https://groups.google.com/forum/#!forum/ddf-developers .
(please choose ONLY two committers from below, delete the rest)
@jlcsmith
@millerw8
#### How should this be tested? (List steps with links to updated documentation)
Full build should suffice
#### What are the relevant tickets?
[DDF-3511](https://codice.atlassian.net/browse/DDF-3511)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by mackncheesiest** [comment]
build now

File: catalog/transformer/catalog-transformer-service-atom/pom.xml [hunk 1]
```diff
@@ -112,7 +112,9 @@
                             com.ctc.wstx.stax, <!-- needed by FOMWriter -->
                             *
                         </Import-Package>
-                        <Export-Package/>
+                        <Export-Package>
+                            ddf.catalog.transformer.response.query.atom
```
Comment (coyotesqrl) [inline catalog/transformer/catalog-transformer-service-atom/pom.xml]: ❓ Shouldn't this be removed from the `Private-Package` declaration if it's being exported?

And _why_ is it being exported?

Comment (bdeining) [reply catalog/transformer/catalog-transformer-service-atom/pom.xml]: It's being exported so `AtomTransformer` can be instantiated in a downstream project with different properties

Comment (bdeining) [reply catalog/transformer/catalog-transformer-service-atom/pom.xml]: Removed it from `Private-Package`

Comment (coyotesqrl) [reply catalog/transformer/catalog-transformer-service-atom/pom.xml]: So it's being constructed on the fly and not retrieved as a service implementation?

Or is it being retrieved and then having one of its four service properties updated on the fly? If the latter, wouldn't it be easy enough to replace this bundle with one that updates the blueprint file, and exposes this service implementation?

Comment (bdeining) [reply catalog/transformer/catalog-transformer-service-atom/pom.xml]: It's being constructed as a new bean in a downstream project and then published as an additional service to the system there.

**Comment by coyotesqrl** [comment]
build now

**Pull request merged**